packages/rpc/lib/src/utils/unix_rpc_client.dart

```diff
@@ -29,7 +29,7 @@ class UnixRPCClient extends ServerProxyBase {
                 error["code"], error["message"], error["data"]);
             completer.completeError(exception);
           } else {
-            completer.complete(utf8.decode(event));
+            completer.complete(utf8.decode(event).toString());
```

Comment:
```suggestion
            completer.complete(utf8.decode(event));
```

Why it is the same as before, why do we need a `toString()`?